src/main/java/net/smartcosmos/dto/metadata/MetadataCreate.java

```diff
@@ -0,0 +1,37 @@
+package net.smartcosmos.dto.metadata;
+
```

Comment:
@andreas you are correct.  I mixed my code reviews and thought these were being used in the REST api